pkg/riff/commands/credential_apply.go

```diff
@@ -154,9 +162,8 @@ func NewCredentialApplyCommand(c *cli.Config) *cobra.Command {
 	cmd.Flags().StringVar(&opts.GcrTokenPath, cli.StripDash(cli.GcrFlagName), "", "path to Google Container Registry service account token `file`")
 	cmd.Flags().StringVar(&opts.Registry, cli.StripDash(cli.RegistryFlagName), "", "registry `url`")
 	cmd.Flags().StringVar(&opts.RegistryUser, cli.StripDash(cli.RegistryUserFlagName), "", "`username` for a registry, the password must be provided via stdin")
+	cmd.Flags().StringVar(&opts.DefaultImagePrefix, cli.StripDash(cli.DefaultImagePrefixFlagName), "", fmt.Sprintf("use this `registry` as the default for built images, implies %s", cli.SetDefaultImagePrefixFlagName))
```

Comment:
we should change `registry` to be either `prefix` or possibly `registry-prefix` since it is the registry + / + user or project like `--default-image-prefix harbor.springdeveloper.com/dev`

Reply:
We need to dispel the notion that the repository has anything to do with the username to authenticate the request. In some cases it does, but more often than not, there is no connection.

I agree that `registry` is the wrong phrase here since the value is for a `repository` prefix.